spec/features/users_can_view_programme_level_activities_spec.rb

```diff
@@ -39,6 +45,7 @@
       end
 
       page_displays_an_activity(activity_presenter: ActivityPresenter.new(programme_activity))
+      expect(page).to_not have_content t("default.button.download_as_xml")
```

Comment:
Is there a reason for not separating this into its own `it` block like for BEIS users?

Reply:
Test suite speed 😬 Building up the context twice didn't seem worth it. Of course, then comes the question of why was it worth for the other cases...